server/seccomp/seccomp.go

```diff
@@ -6,13 +6,30 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"syscall"
 
 	"github.com/docker/docker/pkg/stringutils"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/opencontainers/runtime-tools/generate"
 	libseccomp "github.com/seccomp/libseccomp-golang"
 )
 
+// IsEnabled returns true if seccomp is enabled for the host.
+func IsEnabled() bool {
+	// seccompModeFilter refers to the syscall argument SECCOMP_MODE_FILTER.
+	const seccompModeFilter = uintptr(2)
+
+	var enabled bool
```

Comment:
I think we should avoid using var unless required (as a style). @apilloud this is a minor nit, but can you change this to `enabled := false`